Assets/Scripts/Layers/LayerTypes/ReferencedLayerData.cs

```diff
@@ -80,7 +80,7 @@ private void OnParentChanged()
             Reference.OnProxyTransformParentChanged();
         }
 
-        private void OnSiblingIndexOrParentChanged(int newSiblingIndex)
+        private void OnSiblingIndexOrParentChanged(int newSiblingIndex, LayerData data)
```

Comment:
Dit is niet nodig toch? dit is een private functie? en hieronder lijkt het niet eens gebruikt te worden